src/Scripting/Core/ScriptOptions.cs

```diff
@@ -54,6 +53,7 @@ private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
                 "System.Reflection.Extensions",
                 "System.Reflection.Primitives",
                 "System.Runtime",
+                "System.Runtime.Extensions",
```

Comment:
Needed for `dynamic`

---
In reply to: [110797820](https://github.com/dotnet/roslyn/pull/18471#discussion_r110797820) [](ancestors = 110797820)